### ISSUE TITLE
RTP on ZNPC actions will cause exception

### ISSUE THREAD

**Issue by pwlgrzs** [SuperRonanCraft/BetterRTP#64]
**Plugin Version**:
3.1.0-1

https://www.spigotmc.org/resources/znpcs-1-8-1-16-bungeecord-serversnpcs-open-source.80940/

**Describe The Bug**:
Adding ZNPC console action in format 'rpt player %player_name%' causes exception.
%player_name% is PlaceholderAPI format for current player.

**How To Reproduce**:
Right-click ZNPC with the action added.

**Expected Behavior**:
BetterRPT will teleport user as if /rtp player PLAYER was issued.

**Screenshots/Error Log**:
[11:31:14] [Server thread/WARN]: [ServersNPC] Task #98397 for ServersNPC v3.1 generated an exception
org.bukkit.command.CommandException: Unhandled exception executing command 'rtp' in plugin BetterRTP v3.1.0-1
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[patched_1.16.5.jar:git-Paper-584]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159) ~[patched_1.16.5.jar:git-Paper-584]
	at org.bukkit.craftbukkit.v1_16_R3.CraftServer.dispatchCommand(CraftServer.java:810) ~[patched_1.16.5.jar:git-Paper-584]
	at org.bukkit.Bukkit.dispatchCommand(Bukkit.java:776) ~[patched_1.16.5.jar:git-Paper-584]
	at ak.znetwork.znpcservers.npc.enums.NPCAction.run(NPCAction.java:53) ~[?:?]
	at ak.znetwork.znpcservers.user.ZNPCUser$ZNPCSocketDecoder.lambda$decode$1(ZNPCUser.java:186) ~[?:?]
	at org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftTask.run(CraftTask.java:100) ~[patched_1.16.5.jar:git-Paper-584]
	at org.bukkit.craftbukkit.v1_16_R3.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:468) ~[patched_1.16.5.jar:git-Paper-584]
	at net.minecraft.server.v1_16_R3.MinecraftServer.b(MinecraftServer.java:1413) ~[patched_1.16.5.jar:git-Paper-584]
	at net.minecraft.server.v1_16_R3.DedicatedServer.b(DedicatedServer.java:418) ~[patched_1.16.5.jar:git-Paper-584]
	at net.minecraft.server.v1_16_R3.MinecraftServer.a(MinecraftServer.java:1328) ~[patched_1.16.5.jar:git-Paper-584]
	at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1116) ~[patched_1.16.5.jar:git-Paper-584]
	at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:289) ~[patched_1.16.5.jar:git-Paper-584]
	at java.lang.Thread.run(Unknown Source) [?:?]
Caused by: java.lang.ClassCastException: class com.destroystokyo.paper.console.TerminalConsoleCommandSender cannot be cast to class org.bukkit.entity.Player (com.destroystokyo.paper.console.TerminalConsoleCommandSender and org.bukkit.entity.Player are in unnamed module of loader 'app')
	at me.SuperRonanCraft.BetterRTP.references.worlds.WorldPlayer.getPlayer(WorldPlayer.java:107) ~[?:?]
	at me.SuperRonanCraft.BetterRTP.references.depends.DepEconomy.hasBalance(DepEconomy.java:57) ~[?:?]
	at me.SuperRonanCraft.BetterRTP.player.rtp.RTP.start(RTP.java:252) ~[?:?]
	at me.SuperRonanCraft.BetterRTP.player.commands.Commands.tp(Commands.java:160) ~[?:?]
	at me.SuperRonanCraft.BetterRTP.player.commands.Commands.tp(Commands.java:147) ~[?:?]
	at me.SuperRonanCraft.BetterRTP.player.commands.Commands.tp(Commands.java:142) ~[?:?]
	at me.SuperRonanCraft.BetterRTP.player.commands.types.CmdPlayer.execute(CmdPlayer.java:27) ~[?:?]
	at me.SuperRonanCraft.BetterRTP.player.commands.Commands.commandExecuted(Commands.java:53) ~[?:?]
	at me.SuperRonanCraft.BetterRTP.BetterRTP.onCommand(BetterRTP.java:57) ~[?:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[patched_1.16.5.jar:git-Paper-584]
	... 13 more

**Additional Context**:

**Comment by pwlgrzs** [comment]
I think %player_name% placeholder is not recognized. I changed ZNPC action to rtp player @p and I would "[BetterRTP] Usage: /rtp player <player> [world] [biome1, biome2]" in the console, so ZNPC is issuing the console command correctly. I am reporting here because a different plugin which I don't want to use for another reason was recognizing placeholder so I came to conclusion it might be on BRTP end.
